### ISSUE TITLE
Illegal reflective access operation

### ISSUE THREAD

**Issue by dievsky** [JetBrains-Research/viktor#12]
This is what I've got from a JAR using the latest `viktor` (0.5.2):
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by org.jetbrains.bio.viktor.ResourceLibrary (file:<path-to-jar>) to field java.lang.ClassLoader.usr_paths
WARNING: Please consider reporting this to the maintainers of org.jetbrains.bio.viktor.ResourceLibrary
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
```
From a quick Google search, this seems to be related to Java 9+. The JAR was indeed running on
```bash
$ java -version
openjdk version "11.0.1" 2018-10-16 LTS
OpenJDK Runtime Environment Zulu11.2+3 (build 11.0.1+13-LTS)
OpenJDK 64-Bit Server VM Zulu11.2+3 (build 11.0.1+13-LTS, mixed mode)
```

**Comment by dievsky** [comment]
It remains to be seen whether this is overzealousness on JDK 11 part, or whether we're actually accessing something illegal.

**Comment by dievsky** [comment]
The offender is `ResourceLibrary#install` in `Loader.kt`:
```Kotlin
            // See http://stackoverflow.com/a/15409446 for explanation.
            val usrPathsField = ClassLoader::class.java.getDeclaredField("usr_paths")
            usrPathsField.isAccessible = true
```
referring to http://stackoverflow.com/a/15409446 , a trick for changing the native library path in runtime.

**Comment by dievsky** [comment]
Rewritten the native library loading to use absolute paths (since we know them anyway). Will close after I check this is working on all systems as intended.

**Comment by dievsky** [comment]
Fixed in release 0.5.3.
